libraries/TGRSIAnalysis/TCSM/TCSMData.cxx

```diff
@@ -59,59 +59,48 @@ void TCSMData::Clear(Option_t *opt)
 }
 
 /////////////////////////
-void TCSMData::Print(Option_t *opt) 
```

Comment:
The other reason to leave it with the Option is that TCSMData inherits from TGRSIDetectorData I believe. That means the virtual Print(Option) function in TGRSIDetectorData can call it. There may or may not be a reason to do this, but I don't see a reason to not leave ourselves open to this. I will change it and you can tell me to change it back if there is a reason for this.